CMakeLists.txt

```diff
@@ -66,6 +66,16 @@ if (BUILD_KERNELS)
         message(FATAL_ERROR "Failed to find IGA")
     endif()
 
+    find_program(CMC cmc)
+    if (NOT CMC)
+        message(FATAL_ERROR "Failed to find CMC")
+    endif()
+
+    find_program(PYTHON python)
```

Comment:
which python we actually require? 2 or 3? Optimally we would wish to use FindPythonN cmake module (https://cmake.org/cmake/help/v3.12/module/FindPython3.html). The problem is that we will need to bump cmake version up to 3.13 where they appear.